job_templates/leapp_preupgrade.erb

```diff
@@ -0,0 +1,17 @@
+<%#
+kind: job_template
+name: Run preupgrade via Leapp
+job_category: Leapp
+description_format: 'Upgradeability check for RHEL 7 host'
+provider_type: SSH
+feature: leapp_preupgrade
+model: JobTemplate
+%>
+
+<%= render_template 'Check Leapp' %>
+
+leapp preupgrade
+
+echo "===leap_upgrade_report_start==="
+cat /var/log/leapp/leapp-report.json
+rm -f /var/log/leapp/leapp-report.json
```

Comment:
I wonder if it makes sense to leave the JSON behind, just in case operator wants to join and investigate it further.

Reply:
Not big deal, you can probably run this again to see the output.